client/packages/invoices/src/Prescriptions/DetailView/SidePanel/PrescriptionDetailsSection.tsx

```diff
@@ -186,6 +186,13 @@ export const PrescriptionDetailsSectionComponent: FC = () => {
             disabled={isDisabled}
             size="small"
             sx={{ width: 250 }}
+            slotProps={{
+              input: {
+                sx: {
+                  backgroundColor: 'white',
```

Comment:
We generally don't want to refer to colour directly, we use the "theme" colours, which means we can update them at once (e.g. if someone switches to a "Dark" them, this would look completely wrong).

Have a look at some of the other places this is done, usually using the `useTheme` hook. The colour you'd want here is `theme.palette.background.white` :)

Reply:
Same for the other places too